app/templates/ember_widgets/documentation.hbs

```diff
@@ -163,7 +169,7 @@
     </tr>
     <tr>
       <td>fade</td>
-      <td>yes</td>
+      <td>true</td>
       <td><p>If this is yes then the modal fades in/out.</p></td>
```

Comment:
can you change all of these to `if this is true` if you're going to use true/false